src/components/RightPane/CoursePane/CourseRenderPane.js

```diff
@@ -83,6 +89,18 @@ const flattenSOCObject = (SOCObject) => {
     }, []);
 };
 
+const RecruitmentBanner = ({ className }) => (
+    <div className={className}>
+        Interested in web development?
+        <br />
+        <a href="https://forms.gle/v32Cx65vwhnmxGPv8" target="__blank" rel="noopener noreferrer">
+            Join ICSSC and work on AntAlmanac and other projects!
```

Comment:
Should this be "to" instead of "and"? Idk which sounds better.
```suggestion
            Join ICSSC to work on AntAlmanac and other projects!
```

Reply:
I see your point with "to" since it's showing how that is the purpose of joining, but I still think "and" sounds better, although "sounds better" is kinda subjective, so if more people think "and" is better we can change it, but for now, I'll leave it as-is.